consensus/istanbul/backend/engine.go

```diff
@@ -668,6 +653,54 @@ func (sb *Backend) StopAnnouncing() error {
 	return sb.vph.stopThread()
 }
 
+// StartProxyHandler implements consensus.Istanbul.StartProxyHandler
+func (sb *Backend) StartProxyHandler() error {
+	sb.proxyHandlerMu.Lock()
+	defer sb.proxyHandlerMu.Unlock()
+
+	if sb.proxyHandlerRunning {
+		return istanbul.ErrStartedProxyHandler
+	}
+
+	if !sb.IsProxiedValidator() {
+		return istanbul.ErrValidatorNotProxied
+	} else {
+		if sb.config.ProxyInternalFacingNode != nil && sb.config.ProxyExternalFacingNode != nil {
```

Comment:
nit: because the if case returns, we could move this out of an else to make the code a little clear

Reply:
Ya.  Agreed.  I made the requested changes.